src/order-book/generated/models/OrderPostError.ts

```diff
@@ -10,27 +10,30 @@ export type OrderPostError = {
 export namespace OrderPostError {
 
     export enum errorType {
-        DUPLICATE_ORDER = 'DuplicateOrder',
-        INSUFFICIENT_FEE = 'InsufficientFee',
-        INSUFFICIENT_ALLOWANCE = 'InsufficientAllowance',
+        DUPLICATED_ORDER = 'DuplicatedOrder',
+        QUOTE_NOT_FOUND = 'QuoteNotFound',
+        INVALID_QUOTE = 'InvalidQuote',
+        MISSING_FROM = 'MissingFrom',
+        WRONG_OWNER = 'WrongOwner',
+        INVALID_EIP1271SIGNATURE = 'InvalidEip1271Signature',
         INSUFFICIENT_BALANCE = 'InsufficientBalance',
-        INSUFFICIENT_VALID_TO = 'InsufficientValidTo',
-        EXCESSIVE_VALID_TO = 'ExcessiveValidTo',
+        INSUFFICIENT_ALLOWANCE = 'InsufficientAllowance',
         INVALID_SIGNATURE = 'InvalidSignature',
-        TRANSFER_ETH_TO_CONTRACT = 'TransferEthToContract',
+        INSUFFICIENT_FEE = 'InsufficientFee',
+        SELL_AMOUNT_OVERFLOW = 'SellAmountOverflow',
         TRANSFER_SIMULATION_FAILED = 'TransferSimulationFailed',
-        UNSUPPORTED_TOKEN = 'UnsupportedToken',
-        WRONG_OWNER = 'WrongOwner',
-        INVALID_EIP1271SIGNATURE = 'InvalidEip1271Signature',
-        MISSING_FROM = 'MissingFrom',
-        SAME_BUY_AND_SELL_TOKEN = 'SameBuyAndSellToken',
         ZERO_AMOUNT = 'ZeroAmount',
-        UNSUPPORTED_BUY_TOKEN_DESTINATION = 'UnsupportedBuyTokenDestination',
+        INCOMPATIBLE_SIGNING_SCHEME = 'IncompatibleSigningScheme',
+        TOO_MANY_LIMIT_ORDERS_UNSUPPORTED_BUY_TOKEN_DESTINATION = 'TooManyLimitOrders UnsupportedBuyTokenDestination',
```

Comment:
u merged these two lines?

Reply:
These have just been generated straight from the `openapi.yml`.